shell.nix

```diff
@@ -0,0 +1,17 @@
+{ pkgs ? (import <nixpkgs> {}) }:
+let ff-pkg = { buildStackProject, fetchgit, gmp, zlib, qt5, wrapGAppsHook }:
+buildStackProject {
+  name = "ff";
+
+  src = fetchgit {
+    url = "https://github.com/ff-notes/ff";
+    rev = "18492f6067720977c25cc012df88fdeb9ffee5cc";
```

Comment:
Is this repository the right place for this description? Exact revision is confusing me. Wouldn't it be better to put it in maybe Hydra?

Reply:
This section remained from my attempts to write a package, it's not needed there. I remove it.